src/test/scala/com/fulcrumgenomics/alignment/AlignerTest.scala

```diff
@@ -398,6 +447,12 @@ class AlignerTest extends UnitSpec {
     aln shouldBe "|||.|||||||||||||||||"
   }
 
+  "Aligner.align(minScore)" should "return at least the perfect alignment" in {
+    val results = Aligner(5, -4, -5, -3, mode=Glocal).align("ACGTTTGCAT", "ACGTTTGCAT", 20).sortBy(- _.score)
+    results.size should be >= 1
+    results.head.cigar.toString shouldBe "10="
```

Comment:
how about a test case that actually returns >1 result?

Reply:
Added.